Pony/source/Socket.cpp

```diff
@@ -79,58 +67,56 @@ bool Socket::Open(unsigned short port)
 
 #if defined(_WIN32)
     DWORD nonBlocking = 1;
-    if (ioctlsocket( socket, FIONBIO, &nonBlocking) != 0) {
-	printf( "failed to set non-blocking socket\n" );
-	Close();
-	return false;
-    }
+    if (ioctlsocket(m_socket, FIONBIO, &nonBlocking) != 0) {
 #else
     int nonBlocking = 1;
     if (fcntl(m_socket, F_SETFL, O_NONBLOCK, nonBlocking) == -1) {
+#endif
         Close();
         return false;
     }
-#endif
+
     return true;
 }
 
-int Socket::Send(const Address & dst, const void * data, unsigned size)
+int Socket::Send(const Address & dst, const void * const data, const unsigned size)
```

Comment:
Ampersand should be attached to the type name: `Address& dst`

Reply:
Same goes for pointers.

Reply:
No need to use const when the argument is passed by value

Reply:
ok